pkgs/development/coq-modules/math-classes/default.nix

```
@@ -4,8 +4,10 @@ with lib; mkCoqDerivation {
 
   pname = "math-classes";
   inherit version;
-  defaultVersion = if versions.range "8.6" "8.12" coq.coq-version then "8.12.0" else null;
+  defaultVersion = if versions.range "8.11" "8.13" coq.coq-version then "8.13.0" else
+                   if versions.range "8.6"  "8.10" coq.coq-version then "8.12.0" else null;
```

Note that you could also write this with a switch like this:

```suggestion
  defaultVersion = with versions; switch coq.coq-version [
    { case = range "8.11" "8.13"; out = "8.13.0"; }
    { case = range "8.6"  "8.10"; out = "8.12.0"; }
  ] null;
```

Up to you as the two mean the same thing.

---

If you don't mind, I'll keep the if formulation for now. Hopefully the condition can be returned to its previous simple  form some day if https://github.com/coq-community/math-classes/pull/102 is approved.